var/spack/repos/builtin/packages/bash/package.py

```diff
@@ -16,16 +16,31 @@ class Bash(AutotoolsPackage, GNUMirrorPackage):
 
     maintainers = ["adamjstewart"]
 
+    version("5.2", sha256="a139c166df7ff4471c5e0733051642ee5556c1cc8a4a78f145583c5c81ab32fb")
     version("5.1", sha256="cc012bc860406dcf42f64431bcd3d2fa7560c02915a601aba9cd597a39329baa")
     version("5.0", sha256="b4a80f2ac66170b2913efbfb9f2594f1f76c7b1afd11f799e22035d63077fb4d")
     version("4.4", sha256="d86b3392c1202e8ff5a423b302e6284db7f8f435ea9f39b5b1b20fd3ac36dfcb")
     version("4.3", sha256="afc687a28e0e24dc21b988fa159ff9dbcf6b7caa92ade8645cc6d5605cd024d4")
 
     depends_on("ncurses")
+    depends_on("readline@8.2:", when="@5.2:")
```

Comment:
CHANGELOG claims that readline 8.1+ is required in bash 5.2+, but building against 8.1 results in the following error:
```
Undefined symbols for architecture arm64:
  "_rl_set_timeout", referenced from:
      _reset_timeout in libbuiltins.a(read.o)
      _set_readline_timeout in libbuiltins.a(read.o)
  "_rl_trim_arg_from_keyseq", referenced from:
      _bash_execute_unix_command in bashline.o
ld: symbol(s) not found for architecture arm64
```
Building against 8.2 solves the issue.